nixos/modules/services/networking/trickster.nix

```diff
@@ -58,11 +61,19 @@ in
         '';
       };
 
-      origin = mkOption {
+      origin-type = mkOption {
+        type = types.str;
+        default = "prometheus";
```

Comment:
Not sure how to implement this correctly, but something along the lines of
```nix
types.oneOf [ "prometheus" "influxdb" ];
```
would be better IMO.